tests/unit/mesh/test_mesh.cpp

```diff
@@ -181,6 +181,30 @@ TEST_CASE("MakeSimplicial", "[Mesh]")
    REQUIRE(simplex_mesh.GetNE() == orig_mesh.GetNE()*factor);
 }
 
+TEST_CASE("MakeSimplicialSurfaceIn3D", "[Mesh]")
+{
+   // Note: create a 2D surface mesh surface in 3D space dimension composed of a
+   // single quad. The quad is offset in Z, and check that the vertices of the
+   // simplicial mesh are identical.
+   Mesh orig_mesh(2, 3, 1, 0, 3);
+   orig_mesh.AddVertex(Vector({1.0, 1.0, 1.0}));
```

Comment:
Why do they have the same Z coordinate? Would not it be better for a test to use something more "random", so we are sure they are not being mixed up somehow? 🤔